var/spack/repos/builtin/packages/buildah/package.py

```diff
@@ -0,0 +1,33 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Buildah(MakefilePackage):
+    """Lightweight tool for building Open Container Initiative (OCI) container images"""
+
+    homepage = "https://buildah.io/"
+    git = "https://github.com/containers/buildah"
+
+    maintainers("upsj")
+
+    license("Apache-2.0", checked_by="upsj")
+
+    version("1.37.5", commit="5fd40b989860984a00f6fc1539ff53caceca1325")
```

Comment:
Why use the commit and not a sha256?

Reply:
If you're asking, then it's likely missing familiarity with best practices ;)

Reply:
Not required, just curious since *most* people use sha256s.

That said, the commit is still considered a trusted download (https://spack.readthedocs.io/en/latest/packaging_guide.html#trusted-downloads).

Reply:
And I just learned that the archive hash stability issues in GitHub have been resolved, I only heard about the initial issues, not the fix for them

Reply:
There's really not a fix for archive hash instabilities on GitHub. My preferred way of adding versions to new packages is via git commits these days. (Although I'd add both the commit and tag here which speeds up cloning.)